static/src/javascripts/projects/commercial/modules/dfp/prepare-googletag.js

```diff
@@ -123,7 +125,16 @@ export const init = (): Promise<void> => {
                     npaFlag =
                         Object.keys(state.tcfv2.consents).length === 0 ||
                         Object.values(state.tcfv2.consents).includes(false);
-                    canRun = state.tcfv2.consents['1']; // Store and/or access information on a device
+                    canRun = state.tcfv2.vendorConsents[SOURCEPOINT_ID];
+                    if (canRun) {
+                        loadScript(
+                            config.get(
+                                'libs.googletag',
+                                '//www.googletagservices.com/tag/js/gpt.js'
+                            ),
+                            { async: false }
+                        );
+                    }
```

Comment:
This needs to move out of this call, otherwise it never gets loaded in TCFv1.

Reply:
Yap, still a WIP, adding the AB test on a separate commit soon which should solve this

Reply:
@mxdvl have transformed it now to `Ready for Review`